index.html

```diff
@@ -33,6 +34,134 @@ <h2 class="discrptive-info">I’am a Software Developer</h2>
+                        or sign-ups required. has been the industry's standard</p>
+                        <ul>
+                            <li class="list-btn">html</li>
+                            <li class="list-boot">bootstrap</li>
+                            <li class="list-btn">Ruby</li>
+                        </ul>
+                    <button type="button" class="btn1">See Project</button>
+                </div>
+                <div class="btn-list">
+                    <h3>Profesional Art Printing Data</h3>
+                    <p>A daily selection of privately personalized reads; no accounts
+                        or sign-ups required. has been the industry's standard</p>
+                        <ul>
+                            <li class="list-btn">html</li>
+                            <li class="list-boot">bootstrap</li>
+                            <li class="list-btn">Ruby</li>
+                        </ul>
+                    <button type="button" class="btn1">See Project</button>
+                </div>
+                <div class="btn-list">
+                    <h3>Profesional Art Printing Data</h3>
+                    <p>A daily selection of privately personalized reads; no accounts
+                        or sign-ups required. has been the industry's standard</p>
+                        <ul>
+                            <li class="list-btn">html</li>
+                            <li class="list-boot">bootstrap</li>
+                            <li class="list-btn">Ruby</li>
+                        </ul>
+                    <button type="button" class="btn1">See Project</button>
+                </div>
+                <div class="btn-list">
+                    <h3>Profesional Art Printing Data</h3>
+                    <p>A daily selection of privately personalized reads; no accounts
+                        or sign-ups required. has been the industry's standard</p>
+                        <ul>
+                            <li class="list-btn">html</li>
+                            <li class="list-boot">bootstrap</li>
+                            <li class="list-btn">Ruby</li>
+                        </ul>
+                    <button type="button" class="btn1">See Project</button>
+                </div>
+                <div class="btn-list">
+                    <h3>Profesional Art Printing Data</h3>
+                    <p>A daily selection of privately personalized reads; no accounts
+                        or sign-ups required. has been the industry's standard</p>
+                        <ul>
+                            <li class="list-btn">html</li>
+                            <li class="list-boot">bootstrap</li>
+                            <li class="list-btn">Ruby</li>
+                        </ul>
+                    <button type="button" class="btn1">See Project</button>
+                </div>
+            </div>
+        </section>
+        <div class="about">
```

Comment:
- [ ] Kindly add the `section` tag here, As requested by the previous reviewer since it's a different section and for better use of semantic HTML.